Client/src/components/app-sidebar/app-sidebar.css

```diff
@@ -0,0 +1,63 @@
+.side-nav {
+    padding: 2rem;
+    position: relative;
+    height: 100%;
+    width: 250px;
+}
+.side-nav.minimal {
+    width: 50px;
+    padding: 2rem 1rem;
+}
+.side-nav ul {
+    list-style: none;
+}
+.side-nav li {
+    line-height: 1.5;
+    margin-bottom: 0.5rem;
+}
+.side-nav-ul {
+    padding-left: 0;
+}
+.side-nav-ul > li {
```

Comment:
Graag de `li` een class name geven en daarop stylen, zodat we de markup en styling van elkaar losgekoppeld houden (zie CSS style guide).
Geldt ook voor de regels hieronder